docs/user_guide.md

```diff
@@ -4,63 +4,146 @@ This is a pre-release version of the extension and is not intended for general u
 It may be unstable and documentation is limited.
 If you have any questions, please contact us at [hello@open-diffix.org](mailto:hello@open-diffix.org).
 
-# User Guide
+# Extension configuration
 
-This document provides detailed information about the configuration, behavior and recommended usage
-of __Diffix for PostgreSQL__.
+This document provides detailed information about the configuration, behavior and recommended usage of this extension __pg_diffix__.
```

Comment:
"of this extension pg_diffix" sounds a little redundant, I would pick either "of this extension", or "of pg_diffix".